netsuite/10.0/modules/ROOT/pages/netsuite-call-restlets-example.adoc

```diff
@@ -0,0 +1,159 @@
+= Call NetSuite RESTlets Example
+
+This example shows how to call NetSuite RESTlets that are deployed in NetSuite instances to:
+
+* Create a customer record
+* Retrieve a customer record
+* Delete a record
+
+== Prerequisites
+
+To follow these examples, you must have:
+
+* Java 8
+* Anypoint Studio 7.0.x
+* Mule Runtime 4.1.1 or later
+* DataWeave 2.0
+* Access to a NetSuite instance with credentials in the `mule-app.properties` file
+
+[[deploy-a-script-as-restlet]]
+== Deploy a Script as a RESTlet
+
+. Create a sample JavaScript script:
```

Comment:
So this step, specifically, is where we show getting a record, creating a record, validating mandatory record type, and deleting record, as introduced at the beginning?
Note that the list of actions does not quite match.
Also, if this is the user journey, then why the other subtopics? Why aren't they introduced at the beginning as well?